common/include/Pcsx2Defs.h

```diff
@@ -168,7 +168,7 @@ static const int __pagesize = PCSX2_PAGESIZE;
 #define PCSX2_ALIGNED16_EXTERN(x) extern __declspec(align(16)) x
 
 #define __noinline __declspec(noinline)
-#define __threadlocal __declspec(thread)
+#define thread_local __declspec(thread)
```

Comment:
@blazekousuke @KousukeItsagame 
Any update ? If you're too busy, I can still amend your PR.